tools/ipmonitor/README.md

```diff
@@ -0,0 +1,98 @@
+# Mozilla VPN - IPMonitor
+
+IPMonitor is a simple libpcap app to see which packages are sent and received
```

Comment:
The correct networking terminology here is "packets" when referring to network-layer unit of data being sent and received.